### DISCUSSION TITLE
not recording in 6 channels (Monterey)

### DISCUSSION THREAD

**Discussion by k-popfilters** [ExistentialAudio/BlackHole#596] [Q&A]
any work around?

**Comment by devinroth** [comment]
Check your speaker layout in audio midi setup.

**Comment by k-popfilters** [comment]
i did its not working

**Comment by devinroth** [comment]
Are two channels working?

**Comment by k-popfilters** [comment]
yes

**Comment by k-popfilters** [comment]
its only recording only those two channels

**Comment by devinroth** [comment]
BlackHole doesn't treat channels 1-2 any differently than 3+. The issue is either how you're outputting to BlackHole or recording from. Probably a setting in those apps rather than an issue with BlackHole. If 1-2 is working they all are.

**Comment by k-popfilters** [comment]
im using fl studio

**Comment by devinroth** [comment]
I don’t use Fl so can’t help you there. Maybe check some forums.

**Comment by k-popfilters** [comment]
i used audacity and it is the same too

**Comment by devinroth** [comment]
Audacity doesn’t work for channels over 2.

**Comment by k-popfilters** [comment]
what do you use then to record 6 channels

**Comment by devinroth** [comment]
Logic Pro X.

**Comment by k-popfilters** [comment]
hmm okay
